doc/metrics.md

```diff
@@ -0,0 +1,558 @@
+
+To begin, set up an environment in a new directory:
+
+```bash
+mkdir otel-getting-started
+cd otel-getting-started
+npm init -y
+```
+
+Now install Fastify and OpenTelemetry:
+
+```bash
+npm install fastify @opentelemetry/sdk-node @opentelemetry/exporter-prometheus @opentelemetry/auto-instrumentations-node
+```
+
+The `@opentelemetry/sdk-node` and `@opentelemetry/auto-instrumentations-node` will install all the
+necessary packages to start with OpenTelemetry including instrumentation for a wide variety of popular
+packages, such as `http`, `fetch` etc. The package `@opentelemetry/exporter-prometheus` is installed
+to export our collected metrics to Prometheus.
+
+### Create the sample HTTP Server
+
+Create a file `app.js`:
+
+```javaScript
+const api = require('@opentelemetry/api-metrics')
+const opentelemetry = require("@opentelemetry/sdk-node");
+const { PrometheusExporter } = require("@opentelemetry/exporter-prometheus");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
+
+const prometheusExporter = new PrometheusExporter({ startServer: true });
+
+const sdk = new opentelemetry.NodeSDK({
+  // Optional - If omitted, the metrics SDK will not be initialized
+  metricReader: prometheusExporter,
+  // Optional - you can use the metapackage or load each instrumentation individually
+  instrumentations: [getNodeAutoInstrumentations()],
+  // See the Configuration section below for additional  configuration options
+});
+
+// You can optionally detect resources asynchronously from the environment.
+// Detected resources are merged with the resources provided in the SDK configuration.
+sdk.start().then(() => {
+  // Resources have been detected and SDK is started
+  console.log(`SDK started`)
+
+// Start the http server
+  const fastify = require('fastify')({
+    logger: true
+  })
+
+  fastify.get('/', function (request, reply) {
+    reply.send({ hello: 'world' })
+  })
+
+  fastify.listen({ port: 3000 }, function (err, address) {
+    if (err) {
+      fastify.log.error(err)
+      process.exit(1)
+    }
+
+    console.log(`Server is now listening on ${address}`)
+  })
+});
+
+// You can also use the shutdown method to gracefully shut down the SDK before process shutdown
+// or on some operating system signal.
+const process = require("process");
+process.on("SIGTERM", () => {
+  sdk
+    .shutdown()
+    .then(
+      () => console.log("SDK shut down successfully"),
+      (err) => console.log("Error shutting down SDK", err)
+    )
+    .finally(() => process.exit(0));
+});
+```
+
+In the above example we are initializing the Node SDK to enable the Metrics SDK
+and configure it to export the metrics in Prometheus format by registering the
+`PrometheusExporter`.
+
+You can now run the instrument application and it will run the HTTP server on
+port 3000 with command:
+
+```bash
+node app.js
+```
+
+Now when accessing the HTTP server via [http://localhost:3000](http://localhost:3000) you will
```

Comment:
I think this will not fail the linting step. :slightly_smiling_face: 
```suggestion
Now when accessing the HTTP server via <http://localhost:3000> you will
```